### ISSUE TITLE
Boot roms visible in file select

### ISSUE THREAD

**Issue by Jayde6** [MiSTer-devel/Intv_MiSTer#10]
Haven't tried with a merged rom, but using the individual roms they are visible in the game select menu.

**Comment by mario-64** [comment]
I can confirm this issue.

**Comment by KremlingKuthroat19** [comment]
Yes this issue exists. However, the merged bios does fix this issue.
